react/form/src/components/ConferenceAddFormContainer.js

```diff
@@ -21,19 +21,17 @@ class ConferenceAddFormContainer extends PureComponent {
   }
 
   closeNotification() {
-    this.setState({ notificationMessage: null, notificationStatus: null });
+    this.setState({ notificationMessage: null });
   }
 
   async handleSubmit(conference) {
-    const { t } = this.props;
+    const { t, onCreate } = this.props;
     try {
       const createdConference = await apiConferencePost(conference);
-      const { onCreate } = this.props;
       onCreate(createdConference);
-
       this.setState({
         notificationMessage: t('common.dataSaved'),
-        notificationStatus: 'success',
+        notificationStatus: Notification.SUCCESS,
```

Comment:
`notificationStatus` not in the initial state

Reply:
updated